prowler/lib/persistence/manager.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from .sqlite import SQLiteList, SQLiteDict
+
+default_list = SQLiteList
+
+
+def mklist() -> list:
+    """
+    Create a new list with the given name and data.
+    """
+    prowler_db_connection = os.environ.get('PROWLER_DB_CONNECTION')
```

Comment:
If the default value is `in-memory` we'd need to set that default.
```suggestion
    prowler_db_connection = os.environ.get('PROWLER_DB_CONNECTION', "memory://")
```